packages/core/contracts/financial-templates/common/financial-product-libraries/contract-for-difference-libraries/CoveredCallContractForDifferenceFinancialProductLibrary.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
```

Comment:
Odd that it didn't detect this file as moved...

I'm assuming no logic changes or anything?

Reply:
nothing changed. just renamed.